src/data/services/EnterpriseSubsidyApiService.js

```diff
@@ -6,8 +6,20 @@ import { configuration } from '../../config';
 class SubsidyApiService {
   static baseUrl = `${configuration.ENTERPRISE_SUBSIDY_BASE_URL}/api/v1`;
 
+  static subsidyV2Url = `${configuration.ENTERPRISE_SUBSIDY_BASE_URL}/api/v2/subsidies`;
+
   static apiClient = getAuthenticatedHttpClient;
 
+  static fetchCustomerTransactions(customerUuid, subsidyUuid, options = {}) {
+    const queryParams = new URLSearchParams({
+      ...snakeCaseObject(options),
+    });
+    const url = `${SubsidyApiService.subsidyV2Url}/${subsidyUuid}/transactions/?${queryParams.toString()}`;
+    return SubsidyApiService.apiClient({
+      useCache: configuration.USE_API_CACHE,
+    }).get(url, { clearCacheEntry: true });
```

Comment:
nit: fwiw, neither of these API calls to enterprise-subsidy rely on the cached version of the `httpClient` provided by `@edx/frontend-platform`. Not sure why we'd need to pass `useCache` as a result, since its not relevant when using `getAuthenticatedHttpClient`. Might be worth removing here and below since it's not needed and might cause some confusion around whether the endpoints are cached are not.

Reply:
sure!